lib/bolt/bolt_option_parser.rb

```diff
@@ -422,7 +422,8 @@ def initialize(options)
         @options[:boltdir] = path
       end
       define('--configfile FILEPATH',
-             'Specify where to load config from (default: ~/.puppetlabs/bolt/bolt.yaml)') do |path|
+             'Specify where to load config from (default: ~/.puppetlabs/bolt/bolt.yaml). ' \
+             'Parent directory will be used as the Boltdir.') do |path|
```

Comment:
I'm not sure I understand what `parent directory` means here? For example if `bolt.yaml` is in `/tmp/bolt.yaml`, to me I would think the `parent` directory would be `/`? If you were to use `--configfile /tmp/bolt.yaml` you would expect the boltdir to be `/tmp`, not `/`.

Reply:
"Containing directory"? "File's directory"?